### DISCUSSION TITLE
Centralite 3157100 as Router

### DISCUSSION THREAD

**Discussion by ghost** [Koenkk/zigbee2mqtt#13188] [Devices]
I have a Centralite 3157100 thermostate and i have some other device that have low link quality. Is there any way to tell zigbee2mqtt to make it a router, does that device support routing? Im new too the whole Zigbee concept.

**Comment by redpixy** [comment]
what does the device show up as in z2m? If the Device type shows up as "Router" means it supports routing. You may need to re-pair your "end devices" to the zigbee network again.

**Reply by ghost** [reply]
![Screenshot_20220721-144838_Home Assistant](https://user-images.githubusercontent.com/54512754/180303002-a88da914-e082-4721-aae8-8cbee839fef0.jpg)
 
It's shows up as end device. This is a plugged in device I would expect it to be a router.
